cdap-ui/app/cdap/components/DataPrep/store/DataPrepActionCreator.js

```diff
@@ -120,3 +123,52 @@ export function setWorkspace(workspaceId) {
       });
   });
 }
+
+export function getWorkspaceList(workspaceId) {
+  let namespace = NamespaceStore.getState().selectedNamespace;
+
+  return Rx.Observable.create((observer) => {
```

Comment:
There's no subscriber, but thought probably better to return the list just in case